File: Agile.Cloud.Order.Api/src/main/java/com/huijiewei/agile/cloud/order/api/controllers/HomeController.java
package com.huijiewei.agile.cloud.order.api.controllers;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import springfox.documentation.annotations.ApiIgnore;

@Controller
@ApiIgnore
public class HomeController {
    @RequestMapping(value = "/")
    public String actionIndex() {
        return "redirect:/swagger-ui.html";

    }
}

File: Agile.Cloud.Order.Api/src/main/kotlin/com/huijiewei/agile/cloud/order/api/controllers/HomeController.kt
package com.huijiewei.agile.cloud.order.api.controllers

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping
import springfox.documentation.annotations.ApiIgnore

@Controller
@ApiIgnore
open class HomeController {
    @RequestMapping(value = "/")
    open fun actionIndex(): String {
        return "redirect:/swagger-ui.html"
    }
}